packages/flutter_tools/lib/src/build_system/targets/shader_compiler.dart

```diff
@@ -168,6 +169,10 @@ class ShaderCompiler {
       );
     }
 
+    // TODO(bdero): This path is intended to be identical to `impeller/compiler/shader_lib` in the engine.
+    //              Update this path once the shader lib ships in the engine artifacts.
+    final String shaderLibPath = join(dirname(Platform.script.path), '../../shader_lib');
```

Comment:
When this is packaged with the engine artifacts, it can be looked up similarly to how impellerc is looked up above on line 163

Reply:
Done.